pkg/server/routes.go

```diff
@@ -235,8 +235,11 @@ func (s *Server) routes(
 		return coreClient.GetSystemSummary(ctx, cedarcore.SystemSummaryOpts.WithDeactivatedSystems())
 	}
 
-	dataLoaders := dataloaders.NewDataLoaders(store, userSearchClient.FetchUserInfos, getCedarSystems)
-	dataLoaderMiddleware := dataloaders.NewDataLoaderMiddleware(dataLoaders)
+	dataLoadersFN := func() *dataloaders.DataLoaders {
+		return dataloaders.NewDataLoaders(store, userSearchClient.FetchUserInfos, getCedarSystems)
+	}
+
+	dataLoaderMiddleware := dataloaders.NewDataloaderMiddleware(dataLoadersFN)
```

Comment:
```suggestion
	// We need to construct a NEW copy of dataloaders on each request, since dataloaders
	// cache requests indefinitely once constructed (so they need to be unique per context / HTTP request)
	dataLoaderMiddleware := dataloaders.NewDataloaderMiddleware(dataLoadersFN)
```

Reply:
Thinking a comment here would be helpful for folks who might be tempted to build the dataloaders once, or wonder why we need to pass a function that builds the dataloaders

Reply:
agree! thanks!